packages/webapp/src/containers/Finances/ManageCustomRevenueTypes/index.jsx

```diff
@@ -74,16 +74,24 @@ export default function ManageRevenueTypes({ history }) {
       addLinkText={t('SALE.ADD_SALE.ADD_CUSTOM_REVENUE_TYPE')}
       onAddType={onAddType}
       tileData={customTypes}
-      onTileClick={onTileClick}
       formatTileData={(data) => {
-        const { farm_id, revenue_translation_key, revenue_name, revenue_type_id } = data;
+        const {
+          farm_id,
+          revenue_translation_key,
+          revenue_name,
+          revenue_type_id,
+          custom_description,
+        } = data;
 
         return {
           key: revenue_type_id,
-          tileKey: revenue_type_id,
           icon: icons[farm_id ? 'CUSTOM' : revenue_translation_key],
           label: farm_id ? revenue_name : t(`revenue:${revenue_translation_key}.REVENUE_NAME`),
           className: labelIconStyles.boldLabelIcon,
+          description: farm_id
+            ? custom_description
+            : t(`revenue:${revenue_translation_key}.REVENUE_NAME`),
```

Comment:
```suggestion
            : t(`revenue:${revenue_translation_key}.CUSTOM_DESCRIPTION`),
```

Reply:
But also since this component displays only the custom types, should we need the conditionals at all here or just do `description: custom_description`?

Reply:
Good catch, thank you! Yes they will always have a `farm_id` which is why this bug did nothing 🙂 

Actually now that you have pointed it out, I see this whole data object could use a bit of cleaning!